### ISSUE TITLE
Need to review Running java application locally

### ISSUE THREAD

**Issue by TracyRagan** [ortelius/ortelius#961]
This doc in the contributor guide needs to be reviewed for accuracy. Things may have changed since we published. [Running java application locally](https://docs.ortelius.io/guides/contributorguide/application-build-and-containerisation/01_run_java_application/)

**Comment by Jainil2004** [comment]
hi @TracyRagan I would like to contribute and fix this issue. please provide me with some specifics such as what exactly do you want to be reviewed? do you want me to check if the steps are working as expected? or is there something else?

**Comment by JesseK123** [comment]
I have proof read the document. Please review.

**Comment by TracyRagan** [comment]
Were there any changes? It looks like all was accurate?

**Comment by JesseK123** [comment]
Yes there were quite a few changes.  Have they not come through?

**Comment by JesseK123** [comment]
I have made a pull request, which shows all the changes (I think)
